From: Arnd Bergmann <arnd@arndb.de>
Subject: xenbus: add module description
Date: Mon, 24 Mar 2025 18:32:30 +0100

Modules without a description now cause a warning:

WARNING: modpost: missing MODULE_DESCRIPTION() in drivers/xen/xenbus/xenbus_probe_frontend.o

Link: https://lkml.kernel.org/r/20250324173242.1501003-5-arnd@kernel.org
Signed-off-by: Arnd Bergmann <arnd@arndb.de>
Cc: Jeff Johnson <jeff.johnson@oss.qualcomm.com>
Cc: Juegren Gross <jgross@suse.com>
Cc: Masahiro Yamada <masahiroy@kernel.org>
Cc: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
Cc: Stefano Stabellini <sstabellini@kernel.org>
Cc: Stehen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 drivers/xen/xenbus/xenbus_probe_frontend.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/xen/xenbus/xenbus_probe_frontend.c~xenbus-add-module-description
+++ a/drivers/xen/xenbus/xenbus_probe_frontend.c
@@ -513,4 +513,5 @@ static int __init boot_wait_for_devices(
 late_initcall(boot_wait_for_devices);
 #endif
 
+MODULE_DESCRIPTION("Frontend for Xen bus probe");
 MODULE_LICENSE("GPL");
_
